.github/workflows/build.yml

```diff
@@ -157,7 +157,9 @@ jobs:
       run: |
         echo "COUCH_URL=$COUCH_URL HORTI_BUILDS_SERVER=$MARKET_URL/$BUILDS_SERVER"
         COUCH_URL=$COUCH_URL HORTI_BUILDS_SERVER=$MARKET_URL/$BUILDS_SERVER horti --local --install=medic:medic:test-$TRAVIS_BUILD_NUMBER > tests/logs/horti.log &
-    - name: Test it!
+    - name: Non UI Tests!
```

Comment:
I would rename this integration tests to hopefully encapsulate more! Also, I think this is worthy of it's own job.

Reply:
I thought about calling them 'integration' but we have some `api/integration` tests, I wonder if that could get confusing ! Will try to make them a separate job while we decide on the nomenclature